admin/partials/boldgrid-backup-admin-nav.php

```diff
@@ -71,12 +80,22 @@
 
 $markup = '<h2 class="nav-tab-wrapper">';
 foreach ( $navs as $nav ) {
-	$markup .= sprintf(
-		'<a class="nav-tab %1$s" href="%2$s">%3$s</a>',
-		$nav['class'],
-		$nav['href'],
-		$nav['title']
-	);
+	if ( isset( $nav['count'] ) ) {
```

Comment:
Best practices would have us escaping this content at the time it is displayed.

Because of existing usage, it may be easier to escape it here.

For example:
```
$markup .= sprintf(
			'<a class="nav-tab %1$s" href="%2$s">%3$s %4$s</a>',
			esc_attr( $nav['class'] ),
			esc_url( $nav['href'] ),
			esc_html( $nav['title'] ),
			esc_html( $nav['count'] )
		);
```

Reply:
Corrected